AdblockPlusSafari/FilterList+Processing.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * This file is part of Adblock Plus <https://adblockplus.org/>,
+ * Copyright (C) 2006-2015 Eyeo GmbH
+ *
+ * Adblock Plus is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * Adblock Plus is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Adblock Plus.  If not, see <http://www.gnu.org/licenses/&gt.
+ */
+
+#import "FilterList.h"
+
+// Update filter list every 5 days
+extern const NSTimeInterval FilterListsUpdatePeriod;
```

Comment:
I wonder if it should be renamed to `DefaultFilterListsUpdatePeriod` (or something) now. And while we're renaming it, I think `Interval` would be a slightly better term than `Period`, since the former implies recurrence while the latter doesn't.